lib/integrations/drivers/github.rb

```diff
@@ -0,0 +1,37 @@
+### Github integration
+module Integrations::Drivers
+  class Github < Integrations::BaseDriver
+    def self.display_name
+      'Github'
+    end
+
+    def type
+      :github
+    end
+
+    def create_task(error_id)
+      issue = GroupedIssue.find(error_id)
+      website = @integration.integration.website
+      configuration = @integration.integration.configuration
```

Comment:
This is not ideal at all. You can add a method to get this data in the base_driver.